Makefile

```diff
@@ -14,3 +14,7 @@ doc: env
 
 packages: env
 	bin/packages.sh
+
+binary: clean env packages
+	rm -rf dist build
+	pyinstaller binary/binary.spec
```

Comment:
I think we should put this in `cli/Makefile`. Technically we are turn the `dcoscli` package into a native binary.